src/gitea/utils/response.py

```
@@ -0,0 +1,36 @@
+
+def process_response(response: Response) -> tuple[dict[str, Any] | list[dict[str, Any]], int]:
+    """Process a synchronous HTTP response.
+
+    Args:
+        response: The HTTP response object.
+
+    Returns:
+        A tuple containing the response data and status code.
+    """
+    status_code = response.status_code
+    data = response.json() if status_code == 200 else {}  # noqa: PLR2004
```

Ruff (0.14.13)

Unused `noqa` directive (non-enabled: `PLR2004`)

Remove unused `noqa` directive

(RUF100)